apps/web/modules/bookings/views/bookings-single-view.getServerSideProps.tsx

```diff
@@ -150,9 +150,15 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
   const profile = {
     name: eventType.team?.name || eventType.users[0]?.name || null,
     email: eventType.team ? null : eventType.users[0].email || null,
-    theme: (!eventType.team?.name && eventType.users[0]?.theme) || null,
-    brandColor: eventType.team ? null : eventType.users[0].brandColor || null,
-    darkBrandColor: eventType.team ? null : eventType.users[0].darkBrandColor || null,
+    theme: eventType.team
+      ? null
+      : eventType.profile?.organization?.theme ?? eventType.users[0]?.theme ?? null,
+    brandColor: eventType.team
+      ? null
+      : eventType.profile?.organization?.brandColor ?? eventType.users[0].brandColor ?? null,
+    darkBrandColor: eventType.team
+      ? null
+      : eventType.profile?.organization?.darkBrandColor ?? eventType.users[0].darkBrandColor ?? null,
```

Comment:
I think it would be best to abstract the theming logic(including branding colors) out into a  utility like _getBranding_ or similar.
then we could do something like following everywhere.

```
...getBranding({eventType})
```